test/README.md

```
@@ -272,15 +271,14 @@ scripts/drenv-selftest
 
 ## Using the drenv tool
 
-Before running the `drenv` tool you need to activate the virtual
-environment:
+Before running the `drenv` tool you need to activate the virtual environment:
 
 ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)